In [1]:
a = 34
b = 87
c = a + b
print(f"Sum: {c}")

Sum: 121


# Addition operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 34 and 87 respectively.
* Using addition operator `+`, the sum of `a` and `b` is calculated and is assigned to `c`.
* Result is printed as `Sum: 121`.

In [3]:
a = 64
b = 23
c = a - b
print(f"Difference: {c}")

Difference: 41


# Subtraction operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 64 and 23 respectively.
* Using subtraction operator `-`, the difference of `a` and `b` is calculated and is stored in `c`.
* Result is printed as `Difference: 41`.

In [5]:
a = 12
b = 9
c = a * b
print(f"Product: {c}")

Product: 108


# Multiplication operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 12 and 9 respectively.
* Using multiplication operator `*`, the product of `a` and `b` is calculated and is stored in `c`.
* Result is printed as `Product: 108`.

In [6]:
a = 33
b = 11
c = a / b
print(f"Result: {c}")

Result: 3.0


# Division operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 33 and 11 respectively.
* Using division operator `/`, the result of `a` divided by `b` is calculated and is stored in `c`.
* Result is printed as `Result: 3.0`.

In [9]:
a = 41
b = 3
c = a % b
print(f"Remainder: {c}")

Remainder: 2


# Modulus operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 41 and 3 respectively.
* Modulus operator is used to find the remainder after division.
* Using modulus operator `%`, the remainder of `a` divided by `b` is calculated and is stored in `c`.
* Result is printed as `Remainder: 2`.

In [13]:
a = 41
b = 3
c = a // b
print(f"Floor value: {c}")

Floor value: 13


# Floor Division operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 41 and 3 respectively.
* Floor division operator is used to find the nearest ceiling or integer value excluding values after decimal.
* Using floor division operator `//`, the nearest integer value of `a` divided by `b` is calculated and is stored in `c`. Values after decimal are excluded.
* Result is printed as `Floor value: 13`.

In [14]:
a = 3
b = 4
c = a ** b
print(f"Result: {c}")

Result: 81


# Power operator
* `a`, `b`, and `c` are user defined variables.
* `a` and `b` are assigned with integer values 3 and 4 respectively.
* Power operator is used to find power of the given variables.
* Using power operator `**`, the power of `a` raised by `b` ie `a^b` is calculated and is stored in `c`.
* Result is printed as `Result: 81`.